src/Components/IssuesControl.jsx

```diff
@@ -157,10 +157,11 @@ export function Issues() {
       debug().warn('IssuesControl#Issues: 2, no repo defined')
       return
     }
-    const fetchComments = async (selectedIssue) => {
```

Comment:
no longer async method - why?

Reply:
This function block calls `getComments` in `src/utils/GitHub.js`.

The called function blocks (via use of the `await` keyword) until the Github API returns a response.

Given that it is blocking, there is nothing asynchronous about this function that requires it to be tagged with the `async` modifier.